hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterTrash.java

```diff
@@ -189,6 +189,41 @@ public void testMoveToTrashNoMountPoint() throws IOException,
     assertEquals(2, fileStatuses.length);
   }
 
+  @Test
+  public void testMoveToTrashNoMountPointWithKerBoersUser() throws IOException,
+      URISyntaxException, InterruptedException {
+    //Constructs the structure of the KerBoers user name
+    String kerBoersUser = "randomUser/dev@HADOOP.COM";
```

Comment:
Suggest to correct `KerBoers` to `Kerberos`. Both of the following spell issues.

Reply:
Haha, ok